packages/react-storage/src/components/StorageBrowser/composables/ActionStart.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { ButtonElement } from '../context/elements';
+
+export interface ActionStartProps {
+  onClick?: () => void;
+  disabled?: boolean;
+  label?: string;
+  className?: string;
+}
+
+export const ActionStart = ({
+  onClick,
+  disabled,
+  label,
+  className,
```

Comment:
This shouldn't be a prop on the composable layer, just the control. We should just go with something here that's specific to (but generic to all instances of) storage browser action start components e.g.
```ts
  <ButtonElement
    variant="primary"
    className={`${CLASS_BASE}__action-start`}
    onClick={onClick}
    disabled={disabled}
  >
```